mix.exs

```diff
@@ -33,18 +32,15 @@ defmodule EventStore.Mixfile do
   # Type `mix help deps` for examples and options.
   defp deps do
     [
-      {:phoenix, "~> 1.3.0"},
-      {:phoenix_pubsub, "~> 1.0"},
-      {:phoenix_ecto, "~> 3.2"},
+      {:ecto, "~> 2.2"},
       {:postgrex, ">= 0.0.0"},
-      {:gettext, "~> 0.11"},
-      {:cowboy, "~> 1.0"},
+      {:poison, "~> 3.0"},
+      {:ecto_hlclock, git: "git@github.com:tonic-sys/ecto_hlclock.git"},
```

Comment:
We should go ahead and push a v0.1 of this to hex

Reply:
good call. will do